### PR TITLE
Application skeleton

### PR DESCRIPTION
**Pull request by macosgrove**
Jira card [EMBM-2734](https://envato.atlassian.net/browse/EMBM-2734)

We agreed that Sqlite would suit our needs better than mysql2.

I also beefed up (excuse the pun, Cash Cows!) the application skeleton to do some of the things we know it will need to do. I had some CSV processing code lying around so thought I would reuse it here. I've included basic error logging and tests.

### CODE REVIEW

File: spec/integration/example_integration_spec.rb [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
+        expect(exit_code).to eq(0)
+      end
+    end
+
+    context "when a csv file that does not exist is specified" do
+      subject(:command) { run_command "import missing.csv" }
+
+      it 'has an exit code of 1' do
+        command
+        expect(exit_code).to eq(1)
```
Comment (macosgrove) [inline spec/integration/example_integration_spec.rb]: It is supposed to be possible to get the console output from the `console_output` variable, but I played with it for a while and was only ever able to retrieve the last line of output, which is a blank line when an exception is thrown.

File: lib/configuration/logging.rb [hunk 2]
```diff
@@ -3,22 +3,23 @@
...
-  :color_scheme => $stdout.isatty ? 'bright' : nil
+  pattern: "%m\n",
+  color_scheme: $stdout.isatty ? 'bright' : nil
 )
 
 Logging.appenders.stdout(
   'stdout',
-  :levels => :debug,
-  :layout => console_layout,
+  level: :error,
```
Comment (pablolee) [inline lib/configuration/logging.rb]: was the removal of the 's' in 'levels' intentional?

Comment (macosgrove) [inline lib/configuration/logging.rb]: Yes, it is supposed to be `level`

File: spec/integration/example_integration_spec.rb (hunk 1 above)
Comment (salamagd) [inline spec/integration/example_integration_spec.rb]: Are you able to check if the exception is raised instead? Or is `run_command` actually executing the command in a separate process?

**Comment by salamagd** [comment]
:+1: this is awesome, @macosgrove! Many subtasks hit on the head right here 😄

File: spec/integration/example_integration_spec.rb (hunk 1 above)
Comment (macosgrove) [inline spec/integration/example_integration_spec.rb]: I think the GLI gem rescues the exceptions. But I do check for the exception in the Importer spec.

**Pull request merged**